lua/ttt2/libraries/game_effects.lua

```diff
@@ -148,6 +148,75 @@ function gameEffects.RadiusDamage(dmginfo, pos, radius, inflictor)
     end
 end
 
+-- Creates explosion damage in a sphere through walls. Very useful for making explosives that aren't line of sight based.
+-- @param Player dmgowner The player that causes this explosion.
+-- @param Entity source The entity that causes this explosion.
+-- @param number damage The maximum damage done by this explosion.
+-- @param Vector origin The center of the explosion.
+-- @param number outerRadius The outer border for the explosion damage and its falloff.
+-- @param number innerRadius The inner border for the explosion damage where falloff starts.
+-- @param boolean exponentialFalloff If the damage falloff should be FALSE: Linear, TRUE: exponential.
```

Comment:
personally, I'd settle for linear or exponential and then only support one of these two

Reply:
sounds good to me!